src/time_series.cpp

```diff
@@ -0,0 +1,15 @@
+#include "time_series/multiprocesses_time_series.hpp"
```

Comment:
Shouldn't this file rather be called `multiprocesses_time_series.cpp`?

Reply:
+1